packages/next-server/server/next-server.ts

```diff
@@ -122,6 +131,10 @@ export default class Server {
       this.renderOpts.runtimeConfig = publicRuntimeConfig
     }
 
+    if (compress && this.nextConfig.target !== 'serverless') {
+      this.compression = compression() as Middleware
+    }
```

Comment:
Shouldn’t this be disabled by [@now/next](https://github.com/zeit/now-builders/blob/canary/packages/now-next)?

Reply:
Already is! `@now/next` doesn't use `next-server`.